<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>processesinputform</class>
 <widget class="QWidget" name="processesinputform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius:  6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none;
}

QPushButton:default {
    border-color: navy;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="processNameLabel">
            <property name="styleSheet">
             <string notr="true">font: large &quot;Arial&quot;</string>
            </property>
            <property name="text">
             <string>Process Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="processNameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="noOfSegLabel">
          <property name="styleSheet">
           <string notr="true">font: large &quot;Arial&quot;</string>
          </property>
          <property name="text">
           <string>Number of Segments</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="noOfSegSpinBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="addProcessPushButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
	padding: 4px;
    border-radius:  6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
	font: large &quot;Arial&quot;;
	margin-left: 30px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none;
}

QPushButton:default {
    border-color: navy;
}
</string>
          </property>
          <property name="text">
           <string>Add Process</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="segNameLabel">
          <property name="styleSheet">
           <string notr="true">font: large &quot;Arial&quot;</string>
          </property>
          <property name="text">
           <string>Segment Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="segNameLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="segSizeLabel">
          <property name="styleSheet">
           <string notr="true">font: large &quot;Arial&quot;</string>
          </property>
          <property name="text">
           <string>Segment Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="segSizeSpinBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="addSegmentPushButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
	padding: 4px;
    border-radius:  6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
	font: large &quot;Arial&quot;;
	margin-left: 30px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none;
}

QPushButton:default {
    border-color: navy;
}
</string>
          </property>
          <property name="text">
           <string>Add Segment</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="processSegmentTreeWidget">
      <column>
       <property name="text">
        <string>Process ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Process Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Allocated</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Segment ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Segment Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Segment Size</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>220</y>
     <width>131</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="firstFitCheckBox">
      <property name="text">
       <string>First Fit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bestFitCheckBox">
      <property name="text">
       <string>Best Fit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="worstFitCheckBox">
      <property name="text">
       <string>Worst Fit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="allocatePushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-bottom: 10px;
margin-top: 3px;
font: large &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Allocate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deallocatePushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-bottom: 10px;
font: large &quot;Arial&quot;</string>
      </property>
      <property name="text">
       <string>Deallocate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="drawPushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: large &quot;Arial&quot;</string>
      </property>
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="oldProcessTreeWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>631</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Segment ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Segment Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Allocated</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Segment Size</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="oldProcessLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: large &quot;Arial&quot;;
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Old Process Segments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deallocatePushButton_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>680</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">margin-bottom: 10px;
font: large &quot;Arial&quot;</string>
   </property>
   <property name="text">
    <string>Deallocate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
